src/Request/WebhookRequest/UpdateWebhookRequest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Netflie\WhatsAppCloudApi\Request\WebhookRequest;
+
+use Netflie\WhatsAppCloudApi\Request;
+use Netflie\WhatsAppCloudApi\Request\RequestWithBody;
+
+final class UpdateWebhookRequest extends Request implements RequestWithBody
+{
+    /**
+     * @var string WhatsApp webhook uri.
+     */
+    private string $uri;
+
+    /**
+     * @var ?string WhatsApp webhook verify token, if provided.
+     */
+    private ?string $verify_token;
+
+    /**
+     * @var string WhatsApp Number Id from messages will sent.
+     */
+    private string $from_phone_number_id;
+
+    public function __construct(array $uri, ?string $verify_token, string $access_token, string $from_phone_number_id, ?int $timeout = null)
```

Comment:
The `$uri` is expected to be a string, but you set the type to an array.  Verify token is also required and not null-able

Reply:
Whoops. Sorry for those typos.